learn/how-tos/building-pwa-app.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Enabling PWA - Beta"
+id: "building-pwa-app"
+sidebar_label: "Enabling PWA"
+---
+---
+
+## What is PWA (Progressive Web Application)
+
+PWA (Progressive Web Application) are web applications that provide a native mobile app experience. These applications combine great functionalities and enhanced user experience based on the browser’s capabilities, and works on any platform, including desktop and mobile devices. You can develop PWA using web technologies such as JavaScript, HTML, and CSS.
+
+### Advantages of Using PWA
+
+PWAs have many features and benefits that make them more unique compared to traditional web and native apps, including:
+
+- Fully responsive apps and compatible with browsers
+- Connectivity independence
+- App-like interface
+- Push notifications
+- Automatic updates
+- More secure
+- Discoverability and easy installation
+
+These features make PWA unique from native mobile applications and provide more robust features on any platform using any device.
+
+[![](/learn/assets/pwa/advantages.PNG)](/learn/assets/pwa/advantages.PNG)
+
+## PWA Features Supported by WaveMaker
+
+Currently, the follwoing features are implemented in WaveMaker; that includes:
+
+- Add to the home screen
+- Offline capabilities
```

Comment:
We should add more details on what is cached and what happens when a new version is deployed on the cloud.